web/js/map/layerbuilder.js

```diff
@@ -102,6 +109,35 @@ export default function mapLayerBuilder(config, cache, store) {
     store.dispatch(updateLayerCollection(id));
   };
 
+  // called from tileLoadFunction() when a tile returns an error
+  const handleTileError = (tile, layer, sourceURL) => {
+    const state = store.getState();
+    const { selected: reduxDate } = state.date;
+    const { isKioskModeActive } = state.ui;
+
+    const { id, layerPeriod } = layer;
+
+    const isSubdailyLayer = layerPeriod === 'Subdaily';
+
+    if (isKioskModeActive) {
```

Comment:
We can save a (tiny) bit of processing if we check for kiosk mode earlier in this function. 

```
    const state = store.getState();
    const { isKioskModeActive } = state.ui;
    if (isKioskModeActive) {
       const { selected: reduxDate } = state.date;
       const { id, layerPeriod } = layer;
       const isSubdailyLayer = layerPeriod === 'Subdaily';
```